<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeletarCliente</class>
 <widget class="QMainWindow" name="DeletarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_Box">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(255, 245, 207);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 20px;</string>
         </property>
         <property name="text">
          <string>EXCLUIR CLIENTE POR CPF:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_BotaoLineEdit">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_botaoConfirmar">
          <property name="geometry">
           <rect>
            <x>65</x>
            <y>80</y>
            <width>150</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background: white;
border: 1px solid #FFD580;
font-size: 20px;</string>
          </property>
          <property name="text">
           <string>Confirmar</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>200</width>
            <height>35</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFD580;
font-size: 18px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
